### DISCUSSION TITLE
benefit of using other DB provider ?

### DISCUSSION THREAD

**Discussion by Double-Click-ERP** [drakkan/sftpgo#1276] [Q&A] [answered]
Hello ,

what is the benefits of using DB provider other than the default sqlite (PostgreSQL  / MySQL/MariaDB / CockroachDB ) ?

**Comment by drakkan** [comment, marked as answer]
You can share the same data provider among multiple nodes/pods

**Reply by Joshua2504** [reply]
Do you have experiences with sftpgo performance with sqlite and large-scale setups (I consider 300 users as large-scale)?

**Comment by Double-Click-ERP** [comment]
only that ?
